os-jdg7-launch/added/openshift-launch.sh

```diff
@@ -25,8 +25,35 @@ CONFIGURE_SCRIPTS=(
   /opt/run-java/proxy-options
 )
 
+function runServer() {
+  local instanceDir=$1
+
+  echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
+
+  exec $JBOSS_HOME/bin/standalone.sh -c clustered-openshift.xml -bmanagement 127.0.0.1 -Djboss.server.data.dir="$instanceDir" ${JBOSS_HA_ARGS} ${JAVA_PROXY_OPTIONS}
+}
+
+function init_data_dir() {
+  DATA_DIR="$1"
+  if [ -d /tmp/src/data ]; then
+    if [ ! -f "${DATA_DIR}/../data_initialized" ]; then
+      cp -rf /tmp/src/data/* $DATA_DIR
+      touch "${DATA_DIR}/../data_initialized"
+    fi
+  fi
+}
+
 source $JBOSS_HOME/bin/launch/configure.sh
 
-echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
+if [ "$DATAGRID_SPLIT" = "true" ]; then
+  source /opt/partition/partitionPV.sh
 
-exec $JBOSS_HOME/bin/standalone.sh -c clustered-openshift.xml -bmanagement 127.0.0.1 ${JBOSS_HA_ARGS} ${JAVA_PROXY_OPTIONS}
+  DATA_DIR="${JBOSS_HOME}/standalone/data"
```

Comment:
The assemble script will copy the contents into standalone/data so it may be better for this to be a different directory, perhaps standalone/partition_data or a better name?

Apologies for not catching this before.